### ISSUE TITLE
replace 'opening_hours' with 'simple-opening-hours'

### ISSUE THREAD

**Issue by mojoaxel** [wo-ist-markt/wo-ist-markt.github.io#347]
[opening_hours]() is very complex and has dependencies on moment (BIG!), [i18next-client](https://www.npmjs.com/package/i18next-client) and [suncalc](https://www.npmjs.com/package/suncalc).

It looks like [simple-opening-hours](https://www.npmjs.com/package/simple-opening-hours) also does, what we whant but with a much smaler footprint!

**Comment by Kishore-abhimanyu** [comment]
can you assign this?

**Comment by mojoaxel** [comment]
@Kishore-abhimanyu If you want to work on this, just do so and provide a pull-request :+1:

**Comment by Kishore-abhimanyu** [comment]
Raised a pull request. Please review and merge it

**Comment by johnjohndoe** [comment]
:heavy_check_mark: _moment.js_ has been replaced with _day.js_ in #343 
Is replacing _opening_hours.js_ still a topic?

**Comment by mojoaxel** [comment]
> Is replacing opening_hours.js still a topic?

Yes. This still would simplify  things but also remove some functionality.

**Comment by johnjohndoe** [comment]
> ... but also remove some functionality

Ooh. Like what? Do the libraries not provide the same functionality?

**Comment by mojoaxel** [comment]
[simple-opening-hours](https://www.npmjs.com/package/simple-opening-hours#supported-opening_hours-examples) "only supports the human readable parts".

[opening_hours ](https://www.npmjs.com/package/opening_hours) supports [a much more compley syntax](https://wiki.openstreetmap.org/wiki/Key:opening_hours/specification). I think we don't need this over-engineered syntax and should stick to the "human readable" stuff :-)

**Comment by johnjohndoe** [comment]
Sounds reasonable at first hand. It would be interesting to find out **how many** of the current data points cannot be handled by _days.js_. The result would help to support the decision whether a migration make sense.

Related: Yesterday, I reached out to the Berlin Senat to sort out the "sonstige" data points which **cannot** be parsed by _opening_hours.js_. Pending response.

**Comment by johnjohndoe** [comment]
> Related: Yesterday, I reached out to the Berlin Senat to sort out the "sonstige" data points which **cannot** be parsed by _opening_hours.js_. Pending response.

Meanwhile, the number of "sonstige" markets has been reduced for Berlin. Still some to be sorted out.
